### ISSUE TITLE
FR: Hourly resolution

### ISSUE THREAD

**Issue by Achder** [frappe/gantt#230]
Will you consider implementing an hourly view?
We are using frappe gantt in ERPNext and we could use this feature for shiftplans.

**Comment by Achder** [comment]
I have just seen this PR https://github.com/frappe/gantt/pull/142
Can this be merged?

**Comment by jkiss** [comment]
another solution: https://www.npmjs.com/package/@j.k.bi/react-gantt

**Comment by hahusu** [comment]
I reworked it a little and got it going for me.
[See the jsFiddle link](https://stackoverflow.com/questions/68648967/frappe-gantt-error-hour-mode-error-text-attribute-x-expected-length-nan/72151191#72151191)

[Direktlink without stackoverflow](https://jsfiddle.net/hahussu/43e19btw/)

**Comment by fullmooooon** [comment]
> I reworked it a little and got it going for me. Maybe it helps one of you
> 
> [js fiddle example](https://jsfiddle.net/hahussu/43e19btw/)

```
'Hour_upper': this.options.column_width * 30 / 2,
```
=>
```
'Hour_upper': this.options.column_width * 24 / 2,
```
Upper seems to be right to offset 12 units to center the date text.

**Comment by safwansamsudeen** [comment]
Will be implemented in #387.
